### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mrhardikjoshi**
blah

### CODE REVIEW

**Comment by mrhardikjoshi** [comment]
ok testing 1

**Comment by mrhardikjoshi** [comment]
test 2

**Comment by mrhardikjoshi** [comment]
test 3

**Comment by mrhardikjoshi** [comment]
test 4

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This gem provides a way to get equity information, top loosers gainers, realtime pricing and bids from NSE. THIS GEM IS UNDER DEVELOPMENT AT THE MOMENT AND IS UNSTABLE.
 
+
```
Comment (mrhardikjoshi) [inline README.md:5]: test 1 code comment

**Comment by mrhardikjoshi** [comment]
test 5

**Comment by mrhardikjoshi** [comment]
blah

**Comment by mrhardikjoshi** [comment]
test 5

**Comment by mrhardikjoshi** [comment]
test 6

**Comment by h4hardikonly** [comment]
deply app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
deploy app-platform-r19-qe3.coupadev.com

**Comment by h4hardikonly** [comment]
Created a control-next job for deploying this change on QE instance ([Link to the job](https://control-next.coupahost.com/requests/232918))

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
Created a control-next job for deploying this change on QE instance ([Link to the job](https://control-next.coupahost.com/requests/324))

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
Created a control-next job for deploying this change on QE instance ([Link to the job](https://control-next.coupahost.com/requests/324))

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
Created a control-next job for deploying this change on QE instance ([Link to the job](https://control-next.coupahost.com/requests/324))

**Comment by h4hardikonly** [comment]
deploy clone-app-platform-r20-qe7-jz-23654.coupadev.com

**Comment by h4hardikonly** [comment]
PR is against master branch, please directly merge the PR and QE will verify on Dashmaster instance

**Comment by h4hardikonly** [comment]
deploy

**Comment by h4hardikonly** [comment]
deploy

**Comment by mrhardikjoshi** [comment]
We have some issues bootstrapping the new instance for the PR changes. Retrying it again

**Comment by mrhardikjoshi** [comment]
We have some issues bootstrapping the new instance for the PR changes. Retrying it again

**Pull request closed without merging**